benchmarkcommon/src/main/java/com/ss/benchmark/httpclient/common/Payloads.java

```diff
@@ -28,13 +26,6 @@ private static String load(String target) {
         }
     }
 
-    private static String extractfromJar(InputStream input, String target) throws IOException {
-        Path dir = Files.createTempDirectory("tempdir");
-        Path output = dir.resolve(target);
-        Files.copy(input, output);
-        return output.toString();
-    }
-
     public static final String LONG_JSON = load("/long.json");
```

Comment:
let's leave for now

Reply:
There are a lot of these comments.  Are you opposed to the suggestions?  If not, I may do some of them when I do the RxNetty work.  If you are, please let me know now.  No use in doing work that others agree is wrong:)

Reply:
that method can be removed -- it is not used.  The files however are used both by the clients and the mock server so would require more rework than I believe is needed to get our data.  If you manage both sides I'm good with it just don't see the value.

Reply:
actually @crankydillo I don't see this extractFromJar method -- let's discuss tomorrow as I may be missing something

Reply:
I deleted commented because you made a general comment about my responses to your feedback; let's discuss tomorrow.  On this particular item, the files are used on both the client and the mock server so it may not be worth the effort IMO